src/Resources/config/doctrine/BaseMedia.orm.xml

```diff
@@ -7,7 +7,11 @@
         <field name="providerName" column="provider_name" type="string" length="255"/>
         <field name="providerStatus" column="provider_status" type="integer"/>
         <field name="providerReference" column="provider_reference" type="string" length="255"/>
-        <field name="providerMetadata" column="provider_metadata" type="json" nullable="true"/>
+        <field name="providerMetadata" column="provider_metadata" type="json" nullable="true">
+            <options>
+                <option name="jsonb">true</option>
```

Comment:
Should we add tests for this?

Reply:
I don't think it is really easy to test. The problem with the distinct query is only when you do a batch delete.

I think we might start with testing on different databases instead of trying to reproduce the problem that this PR solves.

Anyway we shouldn't block the PR for its test IMO. I guess if it does not break with Sqlite we are fine.

Reply:
I personally don't block the PR for unit/functional test, but for real tests.

I'd like to avoid to release the 4.0 with a major bug for mysql user and then be in the situation where reverting the change introduce a bug for mongo users.